SignalMessaging/utils/OWSAvatarBuilder.h

```diff
@@ -1,24 +1,33 @@
 //
-//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
 //
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class TSThread;
 @class OWSContactsManager;
+@class TSThread;
 @class UIImage;
 
 @interface OWSAvatarBuilder : NSObject
```

Comment:
The goal here was to DRY up the avatar creation logic so that we had consistent colors, gradients, text sizing, etc.